src/modules/trades.ts

```diff
@@ -60,6 +60,7 @@ export namespace trades {
         trade.order = orderId
         trade.gasPrice = txGasPrice
         trade.feeAmount = feeAmount
+        trade.feeAmountUsd = feeAmountUsd
         trade.buyAmountEth = buyAmountEth
         trade.sellAmountEth = sellAmountEth
```

Comment:
I believe for this field it makes sense to also add feeAmountEth, doesn't it?
It wouldn't be practical for tx cost on gchain as we discussed, but here we already have everything needed to calculate that

Reply:
Done! :rocket: